src/text-editor-element.coffee

```diff
@@ -153,8 +153,10 @@ class TextEditorElement extends HTMLElement
     @component.focused() if event.relatedTarget is this
 
   addGrammarScopeAttribute: ->
```

Comment:
Well I started in text-editor. And then moved to text-editor-element. I think you and @kevinsawicki should chat. I can put it wherever it makes sense.